pkg/operator/starter.go

```diff
@@ -243,26 +243,29 @@ func ensureCRDSExist(ctx context.Context, apiextensionsClient apiextensionsclien
 // present, so we employ some cheap / meets min / don't go down the path
 // of shared informers" means for dealing with inadvertent deletes of the configuration configmap
 func ensureConfigurationConfigMapExists(ctx context.Context, kubeClient kubeclient.Interface) error {
-	cmData, err := assets.ReadFile("config_configmap.yaml")
-	if err != nil {
-		return fmt.Errorf("error occurred reading file 'config_configmap.yaml': %s", err)
-	}
-	configMap := &corev1.ConfigMap{}
-	if err := yaml.Unmarshal(cmData, configMap); err != nil {
-		return fmt.Errorf("error occurred unmarshalling file 'config_configmap.yaml': %s", err)
-	}
-	_, err = kubeClient.CoreV1().ConfigMaps(defaultNamespace).Get(ctx, configMap.Name, metav1.GetOptions{})
-	if kerrors.IsNotFound(err) {
-		_, err = kubeClient.CoreV1().ConfigMaps(defaultNamespace).Create(ctx, configMap, metav1.CreateOptions{})
+	cms := []string{"config_configmap.yaml", "sharedconfigmaplist_configmap.yaml", "sharedsecretlist_configmap.yaml"}
+	for _, cm := range cms {
+		cmData, err := assets.ReadFile(cm)
 		if err != nil {
-			return fmt.Errorf("error occurred creating %q ConfigMap: %s", configMap.Name, err)
+			return fmt.Errorf("error occurred reading file %q: %s", cm, err)
+		}
+		configMap := &corev1.ConfigMap{}
+		if err := yaml.Unmarshal(cmData, configMap); err != nil {
+			return fmt.Errorf("error occurred unmarshalling file %q: %s", cm, err)
+		}
+		_, err = kubeClient.CoreV1().ConfigMaps(defaultNamespace).Get(ctx, configMap.Name, metav1.GetOptions{})
+		if kerrors.IsNotFound(err) {
+			_, err = kubeClient.CoreV1().ConfigMaps(defaultNamespace).Create(ctx, configMap, metav1.CreateOptions{})
+			if err != nil {
+				return fmt.Errorf("error occurred creating %q ConfigMap: %s", configMap.Name, err)
+			}
+			klog.Infof("Successfully created ConfigMap %q", configMap.Name)
+		} else {
+			klog.Infof("ConfigMap %q already exists", configMap.Name)
+		}
+		if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
```suggestion
		} else if err != nil {
```

Reply:
I think this line gives unexpected error occured because error is not `nil` and still `kerrors.IsNotFound(err)` is returning `false`. So in that case it should be totally unexpected. 
May be there is code misplacement. I will fixed that.